crates/router/src/connector/mollie/transformers.rs

```diff
@@ -86,17 +99,28 @@ pub struct Address {
     pub country: api_models::enums::CountryAlpha2,
 }
 
+pub struct MollieBrowserInfo {
+    language: String,
+}
+
 impl TryFrom<&types::PaymentsAuthorizeRouterData> for MolliePaymentsRequest {
     type Error = Error;
     fn try_from(item: &types::PaymentsAuthorizeRouterData) -> Result<Self, Self::Error> {
-        let amount = Amount {
+        let amount: Amount = Amount {
```

Comment:
```suggestion
        let amount = Amount {
```

Specifying the type again is not necessary.